.github/workflows/app-tests.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v3
         with:
-          node-version: "18.x"
+          node-version: "lts/Hydrogen"
```

Comment:
are you sure this is required to be updated, doesn't 18.x do the same thing

Reply:
unfortunately, the Workflow script assumes 18.7, so it fails

I can try 18.18 instead if you like.